src/main/java/net/pms/util/BitRateMode.java

```diff
@@ -0,0 +1,52 @@
+package net.pms.util;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import java.util.Locale;
+
+/**
+ * This {@code enum} represents the possible bit rate "modes".
+ */
+public enum BitRateMode {
+
+	/** Constant bit rate mode*/
+	CBR,
+
+	/** Variable bit rate mode */
+	VBR;
+
+	@Override
+	public String toString() {
+		switch (this) {
+			case CBR:
+				return "Constant";
```

Comment:
Ah, those were the days - when you could use the computer for hours *and* get a tan (or, atleast the sunburn). If I didn't know better, I wouldn't believe you. Nothing bad about CRTs though, I still miss them (except for the sunburn) because they could handle any resolution, had better refresh rates etc, But seriously, 14"? I do remember that those were standard back in the monochrome days. I'm sure you can go to the local trash dump and find a bigger screen. The problem isn't the thickness of your glasses, but the low resolution you're probably running. You get very little "screen area" in pixel dimensions, meaning that you can have very little information on the screen. I get claustrophobia just thinking about it.